.github/workflows/automatic-doc-checks.yml

```diff
@@ -0,0 +1,16 @@
+name: Main Documentation Checks
+
+on:
+  - push
+  - pull_request
+  - workflow_dispatch
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true  
+
+jobs:
+  documentation-checks:
+    uses: canonical/documentation-workflows/.github/workflows/documentation-checks.yaml@main
```

Comment:
Note: I do not understand why we need a specialized GitHub action instead of just calling make directly. This is an indirection that just makes it harder to see what is going on, but this is a discussion I should have with the starter pack.

Reply:
> this is a discussion I should have with the starter pack

Yes, please :) It is a bit roundabout way of doing things -- as discussed during the Sprint.